src/wmr-middleware.js

```diff
@@ -53,7 +53,8 @@ export default function wmrMiddleware({
 	aliases,
 	onError = NOOP,
 	onChange = NOOP,
-	plugins
+	plugins,
+	features = { preact: true }
```

Comment:
The defaults here should probably go in normalize-options:
https://github.com/preactjs/wmr/blob/a49b3b1505468db2af490b1ae2def93c474120f8/src/lib/normalize-options.js#L20